packages/mwp-app-render/src/components/Dom.jsx

```diff
@@ -72,13 +73,18 @@ const DOM = props => {
 		? newrelicHTML.replace(scriptPattern, '').replace(/<\/script>$/, '')
 		: false;
 
+	const uxCaptureFilename = require.resolve('ux-capture/js/ux-capture.min.js');
+	const uxCaptureJS = fs.readFileSync(uxCaptureFilename, 'utf-8');
+
 	return (
 		<html lang={htmlLang}>
 			<head>
 				{head.title.toComponent()}
 				{head.meta.toComponent()}
 				{head.link.toComponent()}
 				{head.script.toComponent()}
+				{uxCaptureJS &&
```

Comment:
don't think you need this conditional - the file is required to be there

Reply:
removed. should we handle errors returned by `fs.readFileSync` ?